### DISCUSSION TITLE
Player sync issues when switching instance

### DISCUSSION THREAD

**Discussion by emortaldev** [Minestom/Minestom#613] [Help] [answered]
When player 1 switches gamemode (and other stuff like teams) and then player 2 switches instance to see player 1, player 1 appears to be in creative.

I tried to show this in this video
https://user-images.githubusercontent.com/28760284/151666343-a20dd6b8-8573-4b3c-a1c0-2120f71b5afd.mp4
(or https://cdn.discordapp.com/attachments/715174453398601748/937006893375840296/minestomsync.mp4 if that doesn't load)

It may also happen with entities but I couldn't reproduce

**Comment by TheMode** [comment]
I could not reproduce, can you confirm using the latest commit? I will then need a minimal reproducer.

**Comment by TheMode** [comment, marked as answer]
My bad, I have been able to reproduce and this has been fixed in f9080fa4f86b9ad196231e7ba631ebc293dba1b8
